src/tag/tag.jsx

```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames';
 import Animate from '../animate';
 import Icon from '../icon';
 import { obj, func, support, KEYCODE } from '../util';
+import zhCN from '../locale/zh-cn';
```

Comment:
`import ConfigProvider from '../config-provider';`

Reply:
change line247 into `export default ConfigProvider.config(Tag);`  after doing this, the component switch to a HOC, and thus it is able to change language